changelog/8658.misc.md

```diff
@@ -0,0 +1 @@
+Increase the timeout to run `tests/test_server.py::assert_trained_model` test successfully on Windows. 
```

Comment:
This could be nitpicking now - shouldn't the name of the test in the changelog also change? 😄

Reply:
This is so stupid that the test name has somehow changed 🙈 Probably my mind was occupied with something else at this point.

Reply:
Fixed now.

Reply:
No worries :)